locals.tf

```diff
@@ -61,7 +61,7 @@ locals {
 # https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/227#issuecomment-1097623677
 locals {
   enforcement_mode_default = {
-    enforcement_mode = null
+    enforcement_mode = {}
```

Comment:
@matt-FFFFFF Isn't this now expected to be a bool?

Reply:
Sorry, my mistake it is now being used in a different local that is a map!

Reply:
This is am empty map, used in case of merging when, e.g. there is no `enforcement_mode` defined in the archetype config override. See the other commend I tagged you in